style.css

```diff
@@ -0,0 +1,866 @@
+    margin: 0;
+  }
+
+  .present ul {
+    position: relative;
+    top: 200px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 20px;
+    padding-left: 0;
+  }
+
+  .social i {
+    font-size: 30px;
+    color: #344563;
+  }
+
+  .mob_work {
+    display: grid;
+    justify-items: center;
+    align-items: center;
+    grid-template-columns: 1fr;
+    grid-template-rows: repeat(auto-fit, minmax(300px, 1fr));
+    background-color: #e5e5e5;
+    margin-bottom: 80px;
+    padding: 15px;
+    gap: 80px;
+  }
+
+  .item {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 15px;
+    background-color: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    padding: 10px 10px 18px 10px;
+    width: 92%;
+  }
+
+  h2 {
+    font-weight: 700;
+    font-size: 38px;
+    line-height: 44px;
+    color: #172b4d;
+    margin: 0;
+  }
+
+  .type {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 10px;
+    margin-top: 10px;
+    height: 30px;
+  }
+
+  .languages,
+  .description {
+    margin-bottom: 0;
+  }
+
+  .languages {
+    margin-top: 15px;
+  }
+
+  .description {
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  /* about section css */
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 60px;
+    background-color: #fff;
+    border: 1px solid #dfe1e6;
+    border-top-right-radius: 70px;
+    padding: 0 10px 1px 10px;
+    height: 1263px;
+  }
+
+  .about_me h2 {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+    max-width: 260px;
+  }
+
+  .about_me h3 {
+    color: #7f8cff;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .about_me .social {
+    margin-bottom: 15px;
+    padding: 0;
+  }
+
+  .expertise {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+  }
+
+  .domaine {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-right: 20px;
+    padding-left: 20px;
+  }
+
+  .domaine h2 {
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    color: #000;
+  }
+
+  .list_language {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 10px;
+    margin-left: 10px;
+  }
+
+  .list_language h3 {
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #253858;
+  }
+
+  .lang {
+    display: flex;
+    flex-direction: row;
+    gap: 15px;
+    background: #f7f7f9;
+    border-radius: 8px;
+    padding: 10px;
+    padding-left: 0;
+  }
+
+  .dom2,
+  .dom3 {
+    height: 40px;
+    border-bottom: 1px solid #dfe1e6;
+    padding-bottom: 20px;
+  }
+
+  /* Contact css part */
+
+  .contact {
+    background-color: #396df2;
+    background-image: url(./images/bg_bout.png);
```

Comment:
- [ ] One of the objectives of this project is to understand how to parse a design template precisely. Let's work on some minor details. Kindly update the background image and color to match the design. The color used in the Figma design is `#6070FF`. 🤝️


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/220438454-8609d3fa-a011-44ab-b762-4abaf22ec170.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/220438456-0002c7c1-4cf3-4365-9678-102d25a6ac00.png" width="300">|

Reply:
yeah right